azurelinuxagent/common/utils/networkutil.py

```diff
@@ -225,20 +227,42 @@ def __execute_cmd(cmd):
                                                                                             ustr(error.stderr))
             raise Exception(msg)
 
+    @staticmethod
+    def __execute_check_command(cmd):
+        # Helper to execute a check command, returns True if no exception
```

Comment:
The first line for the comment seems irrelevant here, the 2nd line below kind of captures the gist of what the function is doing.

Reply:
removed